qiita_db/test/test_user.py

```diff
@@ -305,12 +305,6 @@ def test_verify_code(self):
                  WHERE email = 'new@test.bar' AND dflt = true"""
         self.assertEqual(self.conn_handler.execute_fetchone(sql)[0], 2)
 
-        # Make sure new system messages are linked to user
```

Comment:
Agree and more importantly we need to make sure the pathological case doesn't happens. I have created: https://github.com/biocore/qiita/pull/1731